addons/sms/static/tests/sms_widget_test.js

```diff
@@ -1,16 +1,84 @@
 /** @odoo-module **/
 
+import { addFakeModel } from '@bus/../tests/helpers/model_definitions_helpers';
+import { click, getFixture } from "@web/../tests/helpers/utils";
 import FormView from 'web.FormView';
 import ListView from 'web.ListView';
+import { setupViewRegistries } from "@web/../tests/views/helpers";
+import { start, startServer } from '@mail/../tests/helpers/test_utils';
 import testUtils from 'web.test_utils';
-import { makeView, setupViewRegistries } from "@web/../tests/views/helpers";
-import { getFixture } from "@web/../tests/helpers/utils";
 
 const createView = testUtils.createView;
 
-QUnit.module('fields', {
-    beforeEach: function () {
-        this.data = {
+
+
+QUnit.module('fields', (hooks) => {
+
+    let target = undefined;
+
+    addFakeModel('fields.sms.emojis.partner', {
+        message: {string: "message", type: "text"},
+        foo: {string: "Foo", type: "char", default: "My little Foo Value"},
+        mobile: {string: "mobile", type: "text"},
+    });
+    addFakeModel('fields.sms.emojis.visitor', {
+        mobile: {string: "mobile", type: "text"},
+    });
+
+    /**
+     * open the correct view based on inputs
```

Comment:
for consistency (check also other docstring)
```suggestion
     * Open the correct view based on inputs
```